yarn-project/types/src/l2_block_downloader/l2_block_downloader.ts

```diff
@@ -87,7 +87,7 @@ export class L2BlockDownloader {
    * @param timeout - optional timeout value to prevent permanent blocking
    * @returns The next batch of blocks from the queue.
    */
-  public async getL2Blocks(timeout?: number) {
+  public async getBlocks(timeout?: number): Promise<L2Block[]> {
```

Comment:
Should L2Block just be renamed to block? To keep with this naming trend?

Reply:
I think there is some value in having the "L2" in the block type name because it clarifies that it's an L2 block and not an L1 block. I didn't like the getL2Blocks name simply because everywhere else in codebase it's getBlocks.